marketing/v16/insights.go

```diff
@@ -174,6 +178,7 @@ type Insight struct {
 	PublisherPlatform                string                 `json:"publisher_platform"`
 	PlatformPosition                 string                 `json:"platform_position"`
 	Clicks                           uint64                 `json:"clicks,string"`
+	OutboundClicks                   []OutboundClicks       `json:"outbound_clicks,omitempty"`
```

Comment:
Even though I don't see a problem with it I think the `omitempty` is not necessary because it only applies to encoding the struct to json which if I'm right is not done here.